R-package/NAMESPACE

```diff
@@ -26,17 +28,18 @@ export(cb.save.model)
 export(getinfo)
 export(setinfo)
 export(slice)
-export(xgb.Booster.complete)
 export(xgb.DMatrix)
 export(xgb.DMatrix.hasinfo)
 export(xgb.DMatrix.save)
 export(xgb.attr)
 export(xgb.attributes)
 export(xgb.config)
+export(xgb.copy.Booster)
 export(xgb.create.features)
 export(xgb.cv)
 export(xgb.dump)
 export(xgb.gblinear.history)
+export(xgb.get.Booster.nrounds)
```

Comment:
It's no a big issue, but would be great to align the naming with `num_boosted_rounds`:
- Python `num_boosted_rounds`.
- Java `getNumBoostedRound`.
- Scala `getNumBoostedRound`.

Reply:
Changed to `xgb.get.num.boosted.rounds`.